cmd/tipocket/init.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/tipocket/pkg/scaffolds"
+	"github.com/pingcap/tipocket/pkg/scaffolds/file"
+	"github.com/pingcap/tipocket/pkg/scaffolds/model"
+	"github.com/pingcap/tipocket/pkg/scaffolds/template"
+	"github.com/pingcap/tipocket/pkg/scaffolds/template/testcase"
+	testcasecmd "github.com/pingcap/tipocket/pkg/scaffolds/template/testcase/cmd"
+	"github.com/pingcap/tipocket/pkg/scaffolds/template/workflow"
+)
+
+var (
+	caseNameFlag string
+)
+
+func newInitCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "init",
+		Short:   "Initialize a new test case",
+		Example: "",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			r, _ := regexp.Compile("^[a-z][a-z_]*$")
```

Comment:
There should be no `_` in case name, and we should allow `-`.
![image](https://user-images.githubusercontent.com/4962134/108439490-527f6280-728c-11eb-9302-8f2301d13dea.png)

Reply:
resolved on 5ac0fdded38abe17c3ae27f9c7fabac2226932b3